docfiles/style.css

```diff
@@ -383,17 +356,40 @@ svg {
     margin-top: 4rem;
 }
 
-.mainbody {
-    margin-top: 2em;
-    margin-bottom: 2em;
-    padding-left: 2em;
-    padding-right: 2em;
+.article {
+    display: -ms-grid;
+    display: grid;
+    width: 100%;
+    -ms-grid-columns: 7.5% 1fr 1em;
+    grid-template-columns: 7.5% 1fr 1em;
 }
 
-#docs .ui.container.mainbody {
+.article-inner {
+    -ms-grid-column: 2;
+    grid-column: 2;
+    padding-left: 0;
+}
+
+/* Limit width of content so that it is readable */
+#docs .article .article-inner div {
+    max-width: 1100px !important;
```

Comment:
just checking, all of these !importants are still necessary?

Reply:
for now at least, mostly just from fighting with SUI. Should be able to get rid of more in a future pr that moves more of the content into a grid, but will have to see how that works out